build_docs/kubernetes-support.adoc

```diff
@@ -0,0 +1,19 @@
+== Kubernetes support
+
+=== Manually installing CRDS
+
+For rapid testing of RHOAS CLI we can use Operator CRDs in headles mode (without running controller).
+
+----
+minikube start
+cd olm/olm-catalog/rhoas-operator/0.9.0/manifests 
+kubectl apply -f .
+----
+
+=== Testing with running operator
+
+Operator can be started to process all requests created by RHOAS CLI
```

Comment:
Typically this can also be container. WDYT @secondsun

Reply:
What do you mean can be container?